src/shared/telemetry/defaultTelemetryService.ts

```diff
@@ -246,6 +246,11 @@ export class DefaultTelemetryService implements TelemetryService {
 
     private static readEventsFromCache(cachePath: string): TelemetryEvent[] {
         try {
+            if (!fs.existsSync(cachePath)) {
+                getLogger().info(`telemetry cache not found: '${cachePath}'`)
```

Comment:
This could probably be logged at verbose level.

Reply:
Thought about that, but since it was already logged at error level, info seemed like a good compromise--and because I assumed the positive confirmation was wanted for some telemetry-debugging reason. OTOH the user can always adjust the log-level...